modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc

```diff
@@ -84,6 +84,10 @@ After the connector is created:
 * Ensure that there are no errors in logs and in Redpanda Console.
 * Confirm that Redpanda topics are being replicated. You should see messages coming into the topics.
 
+== Using the Connectors API
+When using Connectors API, instead of `source.cluster.sasl.username` and `source.cluster.sasl.password` configuration fields
+you can specify a value for the `source.cluster.sasl.jaas.config` field.
```

Comment:
```suggestion
== Use the Connectors API
When using the Connectors API, instead of specifying `source.cluster.sasl.username` and `source.cluster.sasl.password`, 
you can specify a value for `source.cluster.sasl.jaas.config`.
```